src/sentry/tasks/autofix.py

```diff
@@ -95,3 +102,107 @@ def run_automation_only_task(group_id: int) -> None:
     run_automation(
         group=group, user=AnonymousUser(), event=event, source=SeerAutomationSource.POST_PROCESS
     )
+
+
+@instrumented_task(
+    name="sentry.tasks.autofix.configure_seer_for_existing_org",
+    namespace=issues_tasks,
+    processing_deadline_duration=90,
+    retry=Retry(times=3),
+)
+def configure_seer_for_existing_org(organization_id: int) -> None:
+    """
+    Configure Seer settings for a new or existing organization migrating to new Seer pricing.
+
+    Sets:
+    - Org-level: enable_seer_coding=True - to override old check
+    - Project-level (all projects): seer_scanner_automation=True, autofix_automation_tuning="medium" or "off"
+    - Seer API (all projects): automated_run_stopping_point="code_changes" or "open_pr"
+    """
+    organization = Organization.objects.get(id=organization_id)
+
+    # Set org-level options
+    organization.update_option("sentry:enable_seer_coding", True)
+
+    projects = list(Project.objects.filter(organization_id=organization_id, status=0))
+    project_ids = [p.id for p in projects]
+
+    # If seer is enabled for an org, every project must have project level settings
+    for project in projects:
+        project.update_option("sentry:seer_scanner_automation", True)
+        # If autofix is "off" (the registered default for all projects), keep it off.
```

Comment:
aren't we expecting automation on by default with the new packaging? with this condition, that means we're going to keep it off for most people instead, no?